kratos/includes/model_part_io.h

```diff
@@ -278,9 +309,11 @@ class KRATOS_API(KRATOS_CORE) ModelPartIO : public IO
     void DivideInputToPartitions(SizeType NumberOfPartitions,
                                 GraphType const& rDomainsColoredGraph,
                                 PartitionIndicesType const& rNodesPartitions,
+//                                 PartitionIndicesType const& rGeometriessPartitions,
```

Comment:
commented code, is it still needed?

Reply:
I am guessing as outlook for partitining of geometries
I would leave it with a comment

Reply:
It is a future proof change